var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -40,6 +41,9 @@ def install(self, spec, prefix):
         if '+shared' in self.spec:
             configure_args.append("--enable-shared")
 
+        if not self.compiler.f77 and not self.compiler.fc:
+            configure_args.append('--enable-fortran=no')
```

Comment:
@mathstuf: @davydden: I am not sure the Hypre guys actively maintain the CMake build system, or that it does all that the autotools one does.  It was put there for a customer who used windows, and I think the team mainly updates the autotools one.  So it might be good to go along with what they've got.

@tzanio @rfalgout @ulrikeyang: is this accurate?